test/test_grover_sim.cpp

```diff
@@ -13,14 +13,26 @@ TEST(GroverSimTest, EmulatedOracle14Test) {
 
     EXPECT_EQ(ddsim.getNumberOfOps(), 0);
     EXPECT_EQ(ddsim.getName(), "emulated_grover_14");
+    EXPECT_EQ(ddsim.getOracle(), ddsim.AdditionalStatistics().at("oracle"));
     ASSERT_EQ(ddsim.getPathOfLeastResistance().second.substr(1), ddsim.AdditionalStatistics().at("oracle"));
 }
 
-TEST(GroverSimTest, EmulatedFixedOracleTest) {
+TEST(GroverSimTest, EmulatedFixedOracleFixedSeedTest) {
     GroverSimulator ddsim("0110011", 0);
     ddsim.Simulate(1);
 
     EXPECT_EQ(ddsim.getNumberOfOps(), 0);
     EXPECT_EQ(ddsim.getName(), "emulated_grover_7");
+    EXPECT_EQ(ddsim.getOracle(), ddsim.AdditionalStatistics().at("oracle"));
     ASSERT_EQ(ddsim.getPathOfLeastResistance().second.substr(1), ddsim.AdditionalStatistics().at("oracle"));
 }
+
+TEST(GroverSimTest, EmulatedFixedOracleTest) {
+    GroverSimulator ddsim("0110001");
+    ddsim.Simulate(1);
+
+    EXPECT_EQ(ddsim.getNumberOfOps(), 0);
+    EXPECT_EQ(ddsim.getName(), "emulated_grover_7");
+    EXPECT_EQ(ddsim.getOracle(), ddsim.AdditionalStatistics().at("oracle"));
+    ASSERT_EQ(ddsim.getPathOfLeastResistance().second.substr(1), ddsim.AdditionalStatistics().at("oracle"));
+}
```

Comment:
Long time since I had to comment this somewhere: missing new line at the end of the file.
Pre-commit ;o)